prow/cmd/tide/BUILD

```diff
@@ -1,4 +1,13 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+go_image(
+    name = "tide",
+    base = "@git-base//image",
+    library = ":go_default_library",
+    visibility = ["//visibility:public"],
```

Comment:
Should we add this visibility specifier to all the `go_image`s?

Reply:
I copied this from the `go_binary`, I have no preference.

Reply:
Lets remove the visibility specification then to make this image more consistent with the others. None of our other images are exposed to other bazel packages.